packages/gestalt/src/__snapshots__/Avatar.test.js.snap

```diff
@@ -224,54 +223,61 @@ exports[`Avatar renders the checkmark on verified default 1`] = `
       className="wash"
     />
   </div>
+</div>
+`;
+
+exports[`Avatar renders the correct size - md 1`] = `
+<div
+  className="box circle relative whiteBg"
+  style={
+    Object {
+      "height": 48,
+      "width": 48,
+    }
+  }
+>
   <div
-    className="absolute box"
+    className="Mask circle willChangeTransform"
```

Comment:
What's the `willChangeTransform` for?

Reply:
@btraut it's part of `Mask` - for more info, see https://github.com/pinterest/gestalt/pull/494